internal/dag/httpproxy_processor.go

```diff
@@ -624,6 +624,12 @@ func (p *HTTPProxyProcessor) computeRoutes(
 			namespace = proxy.Namespace
 		}
 
+		if err := includeMatchConditionsValid(include.Conditions); err != nil {
```

Comment:
we should add a test in `builder_test.go` for the changes in this file, there should be example tests in there for prefix matching that can be used as a template

Reply:
in general since codespell/codegen have failed, we aren't seeing the coverage report which I think will show we need some more unit tests for changes in this file